src/coreclr/nativeaot/System.Private.CoreLib/src/System/Runtime/InteropServices/PInvokeMarshal.cs

```diff
@@ -733,5 +734,68 @@ private static unsafe bool CalculateStringLength(byte* pchBuffer, out int ansiBu
         }
 
         #endregion
+
+        public static unsafe object InitializeCustomMarshaller(RuntimeTypeHandle pParameterType, RuntimeTypeHandle pMarshallerType, string cookie, delegate*<string, object> getInstanceMethod)
+        {
+            if (getInstanceMethod == null)
+            {
+                throw new ApplicationException();
+            }
+
+            var marshaller = CustomMarshallerTable.s_customMarshallersTable.GetOrAdd(new CustomMarshallerKey(pParameterType, pMarshallerType, cookie, getInstanceMethod));
+            if (marshaller == null)
+            {
+                throw new ApplicationException();
```

Comment:
What would cause this exception to be thrown?

Reply:
This case is checked here https://github.com/dotnet/runtimelab/blob/861707ef38233883151921bfac679353be4836f3/src/tests/Interop/ICustomMarshaler/Primitives/ICustomMarshaler.cs#L470

Check above in this method is attempt to cover these tests
https://github.com/dotnet/runtimelab/blob/feature/NativeAOT/src/tests/Interop/ICustomMarshaler/Primitives/ICustomMarshaler.cs#L637-L641

But I cannot pass `null` as pointer value using `Emitter`.